src/app/stack/card-details/component-information/component-snippet/component-snippet.component.html

```diff
@@ -120,13 +120,11 @@ <h4>Details of the dependency: <span class="component-name" [innerText]="compone
                 <span class="info-head">Vulnerabilities:</span>
                 <span class="common-container">
                     <span *ngFor="let cve of component?.public_vulnerabilities" class="wrapper">
-
-                        <span *ngIf="cve.severity == 'critical'" class="circle critical" [innerText]="'C'"
-                            title="Critical"></span>
-                        <span *ngIf="cve.severity == 'high'" class="circle high" [innerText]="'H'" title="High"></span>
-                        <span *ngIf="cve.severity == 'medium'" class="circle medium" [innerText]="'M'"
-                            title="Medium"></span>
-                        <span *ngIf="cve.severity == 'low'" class="circle low" [innerText]="'L'" title="Low">
+                        <span class="md-chips-b severity">
+                                <span *ngIf="cve.severity == 'critical'" class="md-chip-b critical" [innerText]="'Critical'"></span>
+                                <span *ngIf="cve.severity == 'high'" class="md-chip-b high" [innerText]="'High'"></span>
+                                <span *ngIf="cve.severity == 'medium'" class="md-chip-b medium" [innerText]="'Medium'"></span>
+                                <span *ngIf="cve.severity == 'low'" class="md-chip-b low" [innerText]="'Low'"></span>
```

Comment:
Can this be written like below?
```suggestion
                                <span [ngClass]="['md-chip-b', cve.severity, 'capitialize']" [innerText]="cve.severity"></span>
```

also you can make use of `first-letter` selector to captialize the innerText.

```css
.capitialize::first-letter {
  text-transform: capitalize;
}
```

Reply:
It does not work, had tried this earlier

Reply:
It is fixed in 168 PR